ConnectorRevit/ConnectorRevit/Entry/App.cs

```diff
@@ -146,21 +147,35 @@ private void ControlledApplication_ApplicationInitialized(object sender, Autodes
       var eventHandler = ExternalEvent.Create(new SpeckleExternalEventHandler(SpeckleRevitCommand.Bindings));
       SpeckleRevitCommand.Bindings.SetExecutorAndInit(eventHandler);
 #else
-      //DUI2 - pre build app, so that it's faster to open up
-      SpeckleRevitCommand2.InitAvalonia();
-      var bindings = new ConnectorBindingsRevit2(AppInstance);
-      bindings.RegisterAppEvents();
-      SpeckleRevitCommand2.Bindings = bindings;
-      SchedulerCommand.Bindings = bindings;
-      OneClickSendCommand.Bindings = bindings;
-      QuickShareCommand.Bindings = bindings;
+        //DUI2 - pre build app, so that it's faster to open up
+        SpeckleRevitCommand2.InitAvalonia();
+        var bindings = new ConnectorBindingsRevit2(AppInstance);
+        bindings.RegisterAppEvents();
+        SpeckleRevitCommand2.Bindings = bindings;
+        SchedulerCommand.Bindings = bindings;
+        OneClickSendCommand.Bindings = bindings;
+        QuickShareCommand.Bindings = bindings;
 
 
 
-      SpeckleRevitCommand2.RegisterPane();
+        SpeckleRevitCommand2.RegisterPane();
```

Comment:
This is a small thing, but it looks like you may have added some indentation accidentally